.travis.yml

```diff
@@ -1,20 +1,30 @@
 language: php
-php:
-   - 5.4
-   - 5.5
-   - 5.6
+
+matrix:
+  include:
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+      env: SCRUTINIZER_REPORT=true
+    - php: 7.0
+  allow_failures:
+    - php: 7.0
+
 before_install:
    - composer self-update
+
 install:
    - composer install --optimize-autoloader --prefer-source --no-interaction
    - php vendor/autoload.php
+
 script:
    - mkdir -p build/logs
    - php vendor/bin/phpcs --standard=psr2 ./src
    - php vendor/bin/phpmd src/ text codesize,controversial,design,naming,unusedcode
    - php vendor/bin/phpunit
-after_script:
-   - php vendor/bin/coveralls
-   ## Scrutinizer
-   - wget https://scrutinizer-ci.com/ocular.phar
-   - php ocular.phar code-coverage:upload --format=php-clover ./build/logs/clover.xml
+
+after_script: >
+  php vendor/bin/coveralls &&
```

Comment:
The PR to remove Coveralls was approved, please update your PR and remove duplicated commit messages.